docker-compose.yaml

```diff
@@ -0,0 +1,101 @@
+services:
+  development:
```

Comment:
I really doubt that docker compose is a useful tool for our purpose at all. But that might be something to clarify.